spec/support/generators.rb

```diff
@@ -0,0 +1,22 @@
+module Pageflow
+  module Specs
+    module Generators
+      module Macros
```

Comment:
Good point. Probably something to consider moving forward. But for now I also prefer consistency.